cardInfoAPI-Level1.yaml

```diff
@@ -0,0 +1,1402 @@
+      tags:
+        - Cards
+      parameters:
+        - in: query
+          name: contract_reference
+          required: false
+          schema:
+            $ref: '#/components/schemas/ContractReference'
+          description: Retrieves all cards related to a contract. Value has to be a functional reference to the contract that can be known by the end customer.
+        - in: query
+          name: person_reference
+          required: false
+          schema:
+            $ref: '#/components/schemas/PersonReference'
+          description: Retrieves all cards related to an individual person/cardholder. Value has to be a functional reference to the person that can be known by the end customer.
+        - in: query
+          name: card_status
+          required: false
+          schema:
+            $ref: '#/components/schemas/CardStatus'
+          description: Retrieves all cards in a certain status. Only one status can be filtered at a time.
+        - in: query
+          name: card_account_reference
+          required: false
+          schema:
+            $ref: '#/components/schemas/CardAccountReference'
+          description: Retrieves all cards related to a card account (not bank account). Value has to be a functional reference to the account that can be known by the end customer.
+        - in: query
+          name: bank_account_reference
+          required: false
+          schema:
+            $ref: '#/components/schemas/BankAccountReference'
+          description: Retrieves all cards related to a bank account (not card account). Value has to be a functional reference to the account that can be known by the end customer and is usually an IBAN.
+        - $ref: '#/components/parameters/limit'
+        - $ref: '#/components/parameters/offset'
+        - $ref: '#/components/parameters/client_in_header'
+        - $ref: '#/components/parameters/correlation_in_header'
+        - $ref: '#/components/parameters/agent_in_header'
+      responses:
+        '200':
+          description: Payment cards were found which match the query parameters. If no cards matched the parameters or access rights are missing, an empty list is returned.
+          headers:
+            X-Correlation-ID:
+              $ref: '#/components/headers/X-Correlation-ID'
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  cardList:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/CardDetails'
+                  _links:
+                    $ref: '#/components/schemas/LinksPagination'
+        '400':
+          $ref: '#/components/responses/standard400'
+        '500':
+          $ref: '#/components/responses/standard500'
+
+  /cards/{cardId}:
+    get:
+      summary: Provides a single card which matches the id.
```

Comment:
What is the added value to the get list?

Reply:
Having the two endpoints is the best practice for the endpoint design in Common API. It supports a wider range of use cases, e.g. for scenarios where the data user already knows the specific card that he wants to retrieve.